src/main/java/pro/sky/courseworktelegrambot/entities/Pet.java

```diff
@@ -0,0 +1,100 @@
+package pro.sky.courseworktelegrambot.entities;
+
+import javax.persistence.*;
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * Object Pet
+ */
+//Класс не абстрактный.
+//Объекты этого класса Spring будет собирать из тела запросов на создание и обновление
+@MappedSuperclass
+public class Pet {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private String name;
+    private String breed;
+    private int age;
+    @Lob
+    private byte[] photo;
+    private boolean isAdopted;
```

Comment:
```suggestion
    private boolean adopted;
```
на is должны начинаться методы, которые возвращают boolean, поля и переменные без is